src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -1839,21 +1842,15 @@ end
                                           boundary_condition::typeof(boundary_condition_slip_wall),
                                           normal_direction::AbstractVector,
                                           direction, equations, dg, indices...)
-    u_rotate = rotate_to_x(u_inner, normal_direction, equations)
+    factor = (normal_direction[1] * u_inner[2] + normal_direction[2] * u_inner[3])
+    u_normal = factor * SVector(normal_direction / norm(normal_direction))
```

Comment:
You need the square of the norm (`norm(normal_direction)^2 = sum(normal_direction^2)`) because you are multiplying twice by the normal vector. I believe the second variant is more efficient because `norm()` takes the square root.
```suggestion
    u_normal = factor * SVector(normal_direction / sum(normal_direction^2))
```

Reply:
Ah true, thanks.